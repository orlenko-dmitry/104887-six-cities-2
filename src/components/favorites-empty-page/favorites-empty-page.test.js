import React from 'react';
import {shallow} from 'enzyme';

import FavoritesEmptyPage from './favorites-empty-page.jsx';

it(`FavoritesEmptyPage rendres correctly`, () => {
  const tree = shallow(<FavoritesEmptyPage />);

  expect(tree).toMatchSnapshot();
});
