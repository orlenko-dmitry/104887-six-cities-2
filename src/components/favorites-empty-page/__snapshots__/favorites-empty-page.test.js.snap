// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesEmptyPage rendres correctly 1`] = `
<div
  className="page page--favorites-empty"
>
  <PageHeader
    user={
      Object {
        "avatarUrl": "/static/avatar/4.jpg",
        "email": "user@gmail.com",
        "id": 1,
        "isPro": false,
        "name": "user",
      }
    }
  />
  <main
    className="page__main page__main--favorites page__main--favorites-empty"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites favorites--empty"
      >
        <h1
          className="visually-hidden"
        >
          Favorites (empty)
        </h1>
        <div
          className="favorites__status-wrapper"
        >
          <b
            className="favorites__status"
          >
            Nothing yet saved.
          </b>
          <p
            className="favorites__status-description"
          >
            Save properties to narrow down search or plan yor future trips.
          </p>
        </div>
      </section>
    </div>
  </main>
</div>
`;
